programs/solrand/src/lib.rs

```diff
@@ -226,7 +187,16 @@ pub struct TransferAuthority<'info> {
     pub system_program: Program<'info, System>,
 }
 
-#[error]
+#[derive(Accounts)]
+pub struct Cancel<'info> {
+    #[account(mut)]
+    pub requester: Account<'info, Requester>,
+    #[account(mut)]
+    pub authority: Signer<'info>,
+    pub system_program: Program<'info, System>,
+}
```

Comment:
I don't see a cancel function for these instructions to be passed into

Reply:
Oops, started to add this. I'll complete it.